File: Assignment___21/Assignment_5.c
#include<stdio.h>
#include<stdlib.h>

void Digits(int Arr[] , int iLength)
{
    int iCnt = 0 , iNo = 0, iSum = 0;
    for(iCnt = 0 ; iCnt < iLength;iCnt++)
    {
        iNo = Arr[iCnt];

        iSum = 0 ;
        while(iNo != 0)
        {
            iSum += iNo % 10 ;
            iNo = iNo / 10;

        }
        
        printf("%d ",iSum);
    }
    printf("\n");
    
  

}
int main()
{
    int iSize = 0 , iCnt = 0 ;
   
    int *ptr = NULL;

    printf("Enter number of Elements :  ");
    scanf("%d",&iSize);

    ptr = (int *)malloc(iSize * sizeof(int));
    if(ptr == NULL)
    {
        printf("Unable to locate memory : \n");
        return -1;
    }

    printf("Enter %d numbers : \n",iSize);
    for(iCnt = 0  ; iCnt < iSize ; iCnt++)
    {
        printf("Enter %d numbers : ",iCnt+1);
        scanf("%d",&ptr[iCnt]);
    }

    Digits(ptr , iSize); 

    free(ptr);

    return 0 ;
}

File: Assignment__18/Assignment_java/Assignment_5.java

import java.util.*;

class Divible
{
    public void Display(int []Arr , int iLength)
    {
        for(int iCnt = 0 ; iCnt < iLength ; iCnt++)
        {
            if(Arr[iCnt] % 11 == 0 )
            {
                System.out.print(Arr[iCnt]+ " ");
            }
        }
        System.out.println();

    }
}
class Assignment_5
{

    public static void main(String A [])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter Number : ");
        int iSize = sobj.nextInt();
    

        int []ptr = new int[iSize];

        System.out.println("Enter "+iSize+" Elements : ");

        for(int iCnt = 0 ; iCnt < iSize; iCnt++)
        {
            System.out.print("Enter elements "+ (iCnt + 1) +" : ");
            ptr[iCnt] = sobj.nextInt();
        }
        Divible dobj = new Divible();
        dobj.Display(ptr , iSize);

    }
}